### ISSUE TITLE
Hierarchical Classes

### ISSUE THREAD

**Issue by tesorrells** [ultralytics/yolov5#5287]
## ❔Question
Is it possible to have hierarchical class labels for training?
## Additional context
For instance 0 is a dog, 1 is a cat, 0.1 is a lab and 1.1 is a tabby? I know it takes class labels in as floats, but I wasn't sure if it was possible or if the loss function would even take it into account.

**Comment by glenn-jocher** [comment]
@tesorrells yes, this is how YOLO9000 works to detect up to 9000 classes:
https://arxiv.org/abs/1612.08242

It's not available by default but of course you can update the architecture and loss function to do the same with YOLOv5, though the changes are not trivial.
